docs/src/components/side-nav/index.tsx

```diff
@@ -25,38 +25,35 @@ export const SideNav: React.FC<Props> = () => {
         <nav>
           <VerticalTabs>
             <VerticalTabs.Group>
-              {items.map(section => {
-                const sectionHeader = (
-                  <VerticalTabs.Tier1.Header>
-                    {section.header.title}
-                  </VerticalTabs.Tier1.Header>
-                )
-
-                return (
-                  <VerticalTabs.Tier1
-                    header={sectionHeader}
-                    collapsible
-                    key={section.header.title}
-                  >
-                    {section.items.map(item => (
-                      <VerticalTabs.Tier2
-                        header={
-                          <VerticalTabs.Tier2.Header
-                            href={item.href}
-                            onClick={(evt: Event) => {
-                              evt.preventDefault()
-                              navigate(item.href)
-                            }}
-                          >
-                            {item.title}
-                          </VerticalTabs.Tier2.Header>
-                        }
-                        key={item.href}
-                      />
-                    ))}
-                  </VerticalTabs.Tier1>
-                )
-              })}
+              {items.map(section => (
+                <VerticalTabs.Tier1
+                  header={
+                    <VerticalTabs.Tier1.Header>
+                      {section.header.title}
+                    </VerticalTabs.Tier1.Header>
+                  }
+                  collapsible
+                  collapsed
```

Comment:
this is the line added ^

Reply:
@jaketrent This takes care of collapsing by default. The full behavior should be that the expanded/collapsed state of the side nav items should be preserved through page reloads, and if I open a url directly to a page, the section of the navigation it's in should be expanded and the nav item should display the selected state. Would you like me to create a new card for that or address in this PR?

Reply:
Definitely a new card.  That was not in the original issue.